dandischema/models.py

```diff
@@ -796,17 +796,15 @@ class Participant(DandiBaseModel):
         nskey="dandi",
     )
     vendor: Optional[Organization] = Field(None, nskey="dandi")
-    age: Optional[PropertyValue] = Field(
-        None,
+    age: PropertyValue = Field(
```

Comment:
Age can be tricky.
* For some species, such as C. Elegens, the maturity of the organism is poorly indicated by the time since birth. They mature at different rates depending on their environment.
* Sharing are for humans can be against HIPAA.
    * Birthdays are considered HIPAA protected. 
    * If you give the session start time and the age in day resolution, the birthday would be recoverable, so this would also be against HIPAA.
    * It's also against HIPAA to provide the age of the participant if it is over 89 years.
* What would be the age of a fetus that was surgically removed from a pregnant mother?